src/main/resources/reference.conf

```diff
@@ -201,10 +237,10 @@ cassandra-query-journal {
   
   # How many events to fetch in one query (replay) and keep buffered until they
   # are delivered downstreams.
-  max-buffer-size = 100
+  max-buffer-size = 10000
```

Comment:
I'm not sure this is wise. The memory consumption might be too much. The idea with reactive streams is to have rather small bounded buffers.